.github/workflows/python-ci.yml

```diff
@@ -24,6 +24,24 @@ on:
     branches:
     - 'main'
   pull_request:
+    paths-ignore:
+    - '.github/ISSUE_TEMPLATE/**'
+    - '.github/workflows/check-md-link.yml'
+    - '.github/workflows/license_check.yml'
+    - '.github/workflows/python-ci-docs.yml'
+    - '.github/workflows/python-integration.yml'
+    - '.github/workflows/python-release.yml'
+    - '.github/workflows/python-release-docs.yml'
+    - '.github/workflows/stale.yml'
```

Comment:
nit: is there a way to say "ignore everything but the current file"?
if a new GitHub Action is added, we'd have to remember to edit this